web3js-ext/example/utils/keystoreV3.js

```diff
@@ -41,7 +41,7 @@ async function main() {
   const encryptedKey2 = await web3.eth.accounts.wallet.encrypt(password2);
 
   // Delete account before adding the same account already existing in the wallet.
-  web3.eth.accounts.wallet.remove(encryptedKey2[0].address);
```

Comment:
Let's un-use wallet in this example

Reply:
Oops, I updated keystoreV3.js too. 
Thank you.